src/test/java/com/synopsys/integration/alert/performance/utility/ExternalAlertRequestUtility.java

```diff
@@ -0,0 +1,82 @@
+package com.synopsys.integration.alert.performance.utility;
+
+import com.synopsys.integration.exception.IntegrationException;
+import com.synopsys.integration.log.IntLogger;
+import com.synopsys.integration.rest.HttpMethod;
+import com.synopsys.integration.rest.HttpUrl;
+import com.synopsys.integration.rest.body.BodyContent;
+import com.synopsys.integration.rest.body.StringBodyContent;
+import com.synopsys.integration.rest.client.IntHttpClient;
+import com.synopsys.integration.rest.request.Request;
+import com.synopsys.integration.rest.response.Response;
+
+public class ExternalAlertRequestUtility implements AlertRequestUtility {
+    private final IntLogger intLogger;
+    private final IntHttpClient client;
+    private final String alertURL;
+
+    public ExternalAlertRequestUtility(IntLogger intLogger, IntHttpClient client, String alertURL) {
+        this.intLogger = intLogger;
+        this.client = client;
+        this.alertURL = alertURL;
+    }
+
+    public void loginToExternalAlert() throws IntegrationException {
+        //String loginBody = "{\"blackDuckUsername\":\"sysadmin\",\"blackDuckPassword\":\"blackduck\"}";
```

Comment:
If you don't envision running these performance tests against older versions then can we remove this comment? 
Otherwise if you want to support older and newer versions of Alert, can you create a request to get the alert version from the about info?  Based on the alert version create the corresponding login JSON payload.  I would consider that an enhancement to be able to create the login payload based on the version.  Not something to hold up this PR right now.